### ISSUE TITLE
encoding issues in EmailMessage.text

### ISSUE THREAD

**Issue by dwrensha** [jparyani/roundcubemail#4]
The value of the [`EmailMessage.text`](https://github.com/sandstorm-io/sandstorm/issues/1148) field in emails produced by this app is malformed in two ways:
1.  It always ends with a `"/r/n./r/n"` sequence, regardless of what text the user enters.
2.  The lines in the rest of the message are dot-escaped --- a practice specific to SMTP that should not be carried over to our Cap'n Proto email interface.

See https://github.com/sandstorm-io/sandstorm/issues/1148  and https://github.com/sandstorm-io/sandstorm/pull/1149 .

**Comment by kentonv** [comment]
Indeed, this has the visual effect of every message send with this app ending with a line containing a "."...
